packages/fast-usdc/test/fixtures.ts

```diff
@@ -31,7 +32,12 @@ export const MockCctpTxEvidences: Record<
       forwardingChannel: 'channel-21',
       recipientAddress:
         receiverAddress ||
-        'agoric16kv2g7snfc4q24vg3pjdlnnqgngtjpwtetd2h689nz09lcklvh5s8u37ek?EUD=osmo183dejcnmkka5dzcu9xw6mywq0p2m5peks28men',
+        encodeAddressHook(
+          'agoric16kv2g7snfc4q24vg3pjdlnnqgngtjpwtetd2h689nz09lcklvh5s8u37ek',
+          {
+            EUD: 'osmo183dejcnmkka5dzcu9xw6mywq0p2m5peks28men',
+          },
```

Comment:
feel free to ignore these, but they stick out at me:

```suggestion
          { EUD: 'osmo183dejcnmkka5dzcu9xw6mywq0p2m5peks28men' },
```